bin/node/runtime/src/lib.rs

```diff
@@ -391,11 +391,21 @@ impl pallet_treasury::Trait for Runtime {
 }
 
 parameter_types! {
-	pub const ContractTransferFee: Balance = 1 * CENTS;
-	pub const ContractCreationFee: Balance = 1 * CENTS;
+	// NOTE Those are significantly lower than than corresponding fees for transferring funds.
+	// Since if we charge the fee in gas there is basically not enough gas even to make one simple
+	// transfer.
+	//
+	// This basically means that in order to transfer funds it would make more sense to use the
+	// contracts module rather than balances module.
+	//
+	// This should be fixed in nearest future by using the synchronous transfer directly. It will
+	// cut fees from the senders account balance directly.
+	pub const ContractTransferFee: Balance = 1000;
+	pub const ContractCreationFee: Balance = 1000;
+	pub const ContractFee: Balance = 1000;
```

Comment:
Regarding static assertions I am not sure. They are established by the downstream module and actually depend on the environment the pallet is embedded into.

Regarding the costs my rough plan is to provide an integration guide that would dive into things like this. Not these values though, since I expect them to be gone as soon as we migrate to transactional storage.